casr/src/bin/casr-ubsan.rs

```diff
@@ -111,6 +115,9 @@ fn extract_warnings(
         }
     }
 
+    // Update counter
+    *counter.write().unwrap() += 1;
```

Comment:
Counter is not increased for errors, so, `log_progress` may hang

Reply:
Done